doc/rule-descriptions.md

```diff
@@ -15,7 +15,7 @@
 | blink | Ensures &lt;blink&gt; elements are not used | cat.time-and-media, wcag2a, wcag222, section508, section508.22.j | true |
 | button-name | Ensures buttons have discernible text | cat.name-role-value, wcag2a, wcag412, section508, section508.22.a | true |
 | bypass | Ensures each page has at least one mechanism for a user to bypass navigation and jump straight to the content | cat.keyboard, wcag2a, wcag241, section508, section508.22.o | true |
-| checkboxgroup | Ensures related &lt;input type=&quot;checkbox&quot;&gt; elements have a group and that that group designation is consistent | cat.forms, best-practice | true |
+| checkboxgroup | Ensures related &lt;input type=&quot;checkbox&quot;&gt; elements have a group and that group designation is consistent | cat.forms, best-practice | true |
```

Comment:
Actually, this is not a typo, perhaps you could change it to

```
Ensures related &lt;input type=&quot;checkbox&quot;&gt; elements have a group and that the group designation is consistent
```

Reply:
Clarification: there are two sentences here, individually, they could read

Ensures related input elements have a group and
Ensures that the group designation is consistent